idd/Energy+.idd.in

```diff
@@ -63383,12 +63383,15 @@ Boiler:Steam,
        \key OtherFuel1
        \key OtherFuel2
   N1 , \field Maximum Operating Pressure
-       \units Kpa
+       \required-field
```

Comment:
All of the test files with this object have 160000 for a value. Could we use that as the default? Also, I can add some code in the getInput for this object which checks and warns if the pressure received is less than 10E+5. This seem like a workable solution?